cluster/gce/gci/mounter/mounter.go

```diff
@@ -68,7 +68,7 @@ func mountInChroot(rootfsPath string, args []string) error {
 	args = append([]string{rootfsPath, mountCmd}, args...)
 	output, err := exec.Command(chrootCmd, args...).CombinedOutput()
 	if err == nil {
-		return err
+		return nil
```

Comment:
Is there any insufficient of the original code?
I don't think the original code is wrong, so....